doc/sphinx/proofs/writing-proofs/rewriting.rst

```diff
@@ -883,14 +781,129 @@ the conversion in hypotheses :n:`{+ @ident}`.
       proof term. See `#12200
       <https://github.com/coq/coq/issues/12200>`_ for more details.
 
-Conversion tactics applied to hypotheses
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+.. _controlling-the-reduction-strategies:
+
+Controlling the reduction strategies and the conversion algorithm
+----------------------------------------------------------------------
+
+Coq provides reduction strategies that the tactics can invoke and two
+different algorithms to check the convertibility of types. The first
+conversion algorithm lazily compares applicative terms while the other
+is a brute-force but efficient algorithm that first normalizes the
+terms before comparing them. The second algorithm is based on a
+bytecode representation of terms similar to the bytecode
+representation used in the ZINC virtual machine :cite:`Leroy90`. It is
+especially useful for intensive computation of algebraic values, such
+as numbers, and for reflection-based tactics. The commands to fine-
+tune the reduction strategies and the lazy conversion algorithm are
+described first.
+
+.. cmd:: Opaque {+ @reference }
+
+   This command accepts the :attr:`global` attribute.  By default, the scope
+   of :cmd:`Opaque` is limited to the current section or module.
+
+   This command has an effect on unfoldable constants, i.e. on constants
+   defined by :cmd:`Definition` or :cmd:`Let` (with an explicit body), or by a command
+   associated with a definition such as :cmd:`Fixpoint`, etc,
+   or by a proof ended by :cmd:`Defined`. The command tells not to unfold the
+   constants in the :n:`@reference` sequence in tactics using δ-conversion (unfolding
+   a constant is replacing it by its definition).
+
+   :cmd:`Opaque` has also an effect on the conversion algorithm of Coq, telling
+   it to delay the unfolding of a constant as much as possible when Coq
+   has to check the conversion (see Section :ref:`conversion-rules`) of two distinct
+   applied constants.
+
+.. cmd:: Transparent {+ @reference }
+
+   This command accepts the :attr:`global` attribute.  By default, the scope
+   of :cmd:`Transparent` is limited to the current section or module.
+
+   This command is the converse of :cmd:`Opaque` and it applies on unfoldable
```

Comment:
```suggestion
   This command is the inverse of :cmd:`Opaque` and it applies on unfoldable
```
or
```suggestion
   This command is the reverse of :cmd:`Opaque` and it applies on unfoldable
```
The "converse" is the reversal of ordering, which is not what is meant here.